src/plugins/data/server/search/search_service.ts

```diff
@@ -145,7 +146,7 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
 
   public setup(
     core: CoreSetup<DataPluginStartDependencies, DataPluginStart>,
-    { bfetch, expressions, usageCollection, taskManager, security }: SearchServiceSetupDependencies
```

Comment:
Guess taskManager could als be removed of SearchServiceSetupDependencies & SearchServiceStartDependencies (not mandatory for this PR, because of scope, just wanted to add)